lib/Models/ResultPendingCatalogItem.js

```diff
@@ -25,12 +25,12 @@ var AsyncFunctionResultCatalogItem = function(terria) {
     knockout.track(this, ['loadingMessage', 'shortReport']);
```

Comment:
Wondering if "shortReport" belongs in `ResultPendingCatalogItem`, now this has been repurposed? I notice that `invokeTerriaAnalyticsService` and `PlacesLikeMeFunction` don't use `shortReport`, so we may be ok just dropping it. @kring?

Similarly, can you change the "typeName" from "Analysis in Progress"?